net/modemmanager/Makefile

```diff
@@ -55,6 +54,24 @@ define Package/modemmanager/description
   Select Utilities/usb-modeswitch if needed.
 endef
 
+define Package/modemmanager-rpcd
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=RPC interface for ModemManager for rpcd
+  URL:=https://www.freedesktop.org/wiki/Software/ModemManager
+  DEPENDS:= \
+	modemmanager \
+	+MODEMMANAGER_WITH_RPCD:lua-cjson
```

Comment:
Is this prefix `MODEMMANAGER_WITH_RPCD` necessary? This is **not** a compilation option as `MODEMMANAGER_WITH_*` which are defined in the [Config.in](https://github.com/openwrt/packages/blob/master/net/modemmanager/Config.in) file.

Reply:
Right you are! Must have been a leftover from experiments. Good catch!